orc8r/cloud/configs/certifier.yml

```diff
@@ -9,3 +9,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+analytics:
+  # Metrics in this Orchestrator configuration should strictly be generic in
+  # nature independent of the type of deployment. It is to be also free of any
+  # user level metrics
+  metrics:
+    # cert metrics
+    cert_life_hours:
```

Comment:
Will prefer the naming as follows -> cert_life_hours -> cert_expires_in_seconds or cert_expires_in_hours